hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/jobhistory/TestHistoryViewerPrinter.java

```diff
@@ -43,6 +46,19 @@
 
   private final String LINE_SEPARATOR = System.lineSeparator();
 
+  private static Locale DEFAULT_LOCALE;
```

Comment:
Sorry I missed in previous review, but do you mind making this a static final variable e.g.
```
private static final Locale DEFAULT_LOCALE = Locale.getDefault();
```
?

Reply:
@liuml07 
I think the static DEFAULT_LOCALE which could be stale during other test suites or settings,
 because, the DEFAULT_LOCALE will be set before all of the tests.

What do you think about?

Reply:
This is only initialized during the static `@BeforeClass setUp()` method, and it will not change during every test cases in this class.